man/EuPathDB.Rd

```diff
@@ -41,18 +41,19 @@ APIs.
 For examples of how EuPathDB can be used to query and interact with
 EuPathDB.org resources, take a look at the vignette:
 \code{browseVignettes(package="EuPathDB")}
+
+Use \code{availableEuPathDB()} to get a vector of available organisms.
 }
 \examples{
 availableEuPathDB
 }
-\author{
-Keith Hughitt
-}
 \seealso{
 \code{\link{AnnotationHub}}
 
 \code{\link{GRanges}}
 
 \url{http://eupathdb.org/eupathdb/}
 }
-
+\author{
+Keith Hughitt
```

Comment:
You should add yourself to the author list here, and in the `DESCRIPTION` file / anywhere else that is relevant.

Reply:
ok